pkg/smb/nodeserver_test.go

```diff
@@ -58,6 +71,12 @@ func TestNodeStageVolume(t *testing.T) {
 		desc        string
 		req         csi.NodeStageVolumeRequest
 		expectedErr testutil.TestError
+
+		// use the below fields only for flaky Windows error messages
+
+		// if true, use substring to match errors
+		isWindowsErrorFlaky bool
+		requiredErrorSubstr string
```

Comment:
@andyzhangx Only for `TestNodeStageVolume`, I have added functionality to check for error messages using substring as some parts of the error messages may change, and are a bit flaky. This way the base error message is verified on Windows to tackle the flakiness.